clidocstool_test.go

```diff
@@ -177,7 +177,7 @@ func init() {
 func TestGenAllTree(t *testing.T) {
 	tmpdir, err := ioutil.TempDir("", "test-gen-all-tree")
 	require.NoError(t, err)
-	defer os.RemoveAll(tmpdir)
+	//defer os.RemoveAll(tmpdir)
```

Comment:
Was this meant to be removed?

(we should also look at updating our tests everywhere to use the `t.Cleanup()` feature - I think `gotest.tools` also can support it automatically for temp-dirs / files etc)

Reply:
> (we should also look at updating our tests everywhere to use the `t.Cleanup()` feature - I think `gotest.tools` also can support it automatically for temp-dirs / files etc)

Oh didn't know about that one.